### PR TITLE
feat: save failed result #ntrn-80

### PR DESCRIPTION
**Pull request by NeverHappened**
https://hadronlabs.atlassian.net/browse/NTRN-80

### CODE REVIEW

**Comment by NeverHappened** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6013284667

File: src/testcases/parallel/subdao.test.ts [hunk 1]
```diff
@@ -147,6 +153,162 @@ describe('Neutron / Subdao', () => {
...
+      );
+      const timelockedProp = await subdaoMember1.supportAndExecuteProposal(
+        proposalId,
+      );
+      expect(timelockedProp.status).toEqual('timelocked');
+      //wait for timelock durations
+      await wait(20);
+      await subdaoMember1.executeTimelockedProposal(proposalId); // should execute no problem
+
+      neutronChain.blockWaiter.waitBlocks(2);
```
Comment (sotnikov-s) [inline src/testcases/parallel/subdao.test.ts:248]: forgotten `await`

File: src/testcases/parallel/subdao.test.ts [hunk 2]
```diff
@@ -147,6 +153,162 @@ describe('Neutron / Subdao', () => {
...
+    });
+
+    let proposalId3: number;
+    test('proposal timelock 3', async () => {
```
Comment (sotnikov-s) [inline src/testcases/parallel/subdao.test.ts:252]: please add a meaningful test case name

File: src/testcases/parallel/subdao.test.ts [hunk 3]
```diff
@@ -147,6 +153,162 @@ describe('Neutron / Subdao', () => {
...
+          subDao.contracts.proposals.single.address,
+          {
+            config: {},
+          },
+        );
+      expect(subdaoConfig.close_proposal_on_execution_failure).toEqual(false);
+
+      //wait for timelock durations
+      await wait(20);
+      // timelocked proposal execution failed due to invalid param value
```
Comment (sotnikov-s) [inline src/testcases/parallel/subdao.test.ts:282]: due to invalid param value  -> due to insufficient funds?

**Review by sotnikov-s — CHANGES_REQUESTED** [review]

**Review by sotnikov-s — APPROVED** [review]

**Comment by pr0n00gler** [comment]
Merged in https://github.com/neutron-org/neutron-integration-tests/pull/220

**Pull request closed without merging**